todoapp/todos/serializers.py

```
@@ -19,3 +21,28 @@ def get_status(self, obj):
+class TodoSerializer(serializers.ModelSerializer):
+    todo_id = serializers.IntegerField(source="id")
+    todo = serializers.CharField(source="name")
+
+    class Meta:
+        model = todos_models.Todo
+        fields = ["todo_id", "todo", "done"]
+
+
+class TodoCreateSerializer(serializers.ModelSerializer):
+    user_id = serializers.PrimaryKeyRelatedField(
+        source="user", queryset=get_user_model().objects.all(), write_only = True
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)